### PR TITLE
Implemented SA1128 codefix

### LINKED ISSUE #1428: Code fix: SA1128 (ConstructorInitializerMustBeOnOwnLine)
Implement a code fix for #1177.

### PR DESCRIPTION
**Pull request by vweijsters**
Fixes #1428

### CODE REVIEW

File: StyleCop.Analyzers/StyleCop.Analyzers/ReadabilityRules/SA1128CodeFixProvider.cs [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.CodeAnalysis;
+    using Microsoft.CodeAnalysis.CodeActions;
+    using Microsoft.CodeAnalysis.CodeFixes;
+    using Microsoft.CodeAnalysis.CSharp;
+    using Microsoft.CodeAnalysis.CSharp.Syntax;
+    using StyleCop.Analyzers.Helpers;
+
+    /// <summary>
```
Comment (sharwell) [inline StyleCop.Analyzers/StyleCop.Analyzers/ReadabilityRules/SA1128CodeFixProvider.cs]: :exclamation: This comment is wrong.

File: StyleCop.Analyzers/StyleCop.Analyzers/ReadabilityRules/SA1128CodeFixProvider.cs [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+    /// </summary>
+    /// <remarks>
+    /// <para>To fix a violation of this rule, add or remove a space after the keyword, according to the description
+    /// above.</para>
+    /// </remarks>
+    [ExportCodeFixProvider(LanguageNames.CSharp, Name = nameof(SA1128CodeFixProvider))]
+    [Shared]
+    internal class SA1128CodeFixProvider : CodeFixProvider
+    {
+        ////private static readonly SA1107FixAllProvider FixAllProvider = new SA1107FixAllProvider();
```
Comment (sharwell) [inline StyleCop.Analyzers/StyleCop.Analyzers/ReadabilityRules/SA1128CodeFixProvider.cs]: :bulb: Remove this line?

**Comment by vweijsters** [comment]
Updated after CR. Refactored FixAll class to use the same mechanism as the one in #1514

File: StyleCop.Analyzers/StyleCop.Analyzers/ReadabilityRules/SA1128CodeFixProvider.cs [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+    using Microsoft.CodeAnalysis.CodeFixes;
+    using Microsoft.CodeAnalysis.CSharp;
+    using Microsoft.CodeAnalysis.CSharp.Syntax;
+    using StyleCop.Analyzers.Helpers;
+
+    /// <summary>
+    /// Implements a code fix for <see cref="SA1128ConstructorInitializerMustBeOnOwnLine"/>.
+    /// </summary>
+    /// <remarks>
+    /// <para>To fix a violation of this rule, add or remove a space after the keyword, according to the description
```
Comment (sharwell) [inline StyleCop.Analyzers/StyleCop.Analyzers/ReadabilityRules/SA1128CodeFixProvider.cs]: :exclamation: This comment is wrong.

Comment (sharwell) [inline StyleCop.Analyzers/StyleCop.Analyzers/ReadabilityRules/SA1128CodeFixProvider.cs]: :arrow_right: I fixed this during the merge.

Comment (vweijsters) [inline StyleCop.Analyzers/StyleCop.Analyzers/ReadabilityRules/SA1128CodeFixProvider.cs]: Thanks :smile:

**Pull request merged**